change/@griffel-react-41ab2ed0-7da1-4045-86ef-915d5b5a7cd0.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "fix: Improve typings for styles object",
+  "packageName": "@griffel/react",
+  "email": "miroslav.stastny@microsoft.com",
+  "dependentChangeType": "patch"
+}
```

Comment:
@layershifter - is it correct that the change entry was generated for anything else than `@griffel/core`?

Reply:
No, can you please delete these changefiles?